HelloMixpanel/HelloMixpanelTests/HelloMixpanelTests.m

```diff
@@ -570,6 +570,21 @@ - (void)testEventTiming {
     XCTAssertNil(p[@"$duration"], @"Tracking the same event should require a second call to timeEvent.");
 }
 
+- (void)testNetworkingWithStress {
+    self.mixpanelWillFlush = NO;
+    stubTrack().andReturn(503);
+    for (int i=1; i<=500; i++) {
```

Comment:
Can we pad the operators and use the typesafe `int`?

`for (NSInteger i = 1; i <= 500; i++) {`